### ISSUE TITLE
Add option to `text` property to avoid text normalization 

### ISSUE THREAD

**Issue by san650** [san650/ember-cli-page-object#158]
In same cases I want to get the exact text that contains an element without the trimming and space normalization we do by default. https://github.com/san650/ember-cli-page-object/blob/754d7a1cd0c6337bb3cd590673b4b215ea3aaee2/test-support/page-object/queries/text.js#L74

It would be great to have a `normalize: false` option to avoid the normalization process

```js
var page = create({
  pre: text('pre', { normalize: false }) 
});

...

andThen(function() {
  assert.equal(page.pre, '  hello world \n\n');
});
```

**Comment by jeradg** [comment]
:+1:

**Comment by vvscode** [comment]
@san650  https://github.com/san650/ember-cli-page-object/pull/161/files

**Comment by san650** [comment]
Fixed by #161

**Comment by san650** [comment]
Thanks @vvscode for contributing!
